### ISSUE TITLE
Does Add-SslCert function work correctly with "netsh show sslcert" command?

### ISSUE THREAD

**Issue by DenVifer** [microsoft/azure-pipelines-extensions#862]
I have a problem with Azre DevOps Release pipeline which is similar to [this](https://github.com/microsoft/azure-pipelines-tasks/issues/13398) issue.

I was investigating this and found quite strange behavior on these lines of the [AppCmdOnTargetMachines](https://github.com/microsoft/azure-pipelines-extensions/blob/master/TaskModules/powershell/TaskModuleIISManageUtility/AppCmdOnTargetMachines.ps1) module:

`
$isItSameBinding = $result.Get(4).Contains([string]::Format("{0}:{1}", $hostname, $port))
`

...

`
$isItSameBinding = $result.Get(4).Contains([string]::Format("{0}:{1}", $ipAddress, $port))
`

...

`
$isItSameCert = $result.Get(5).ToLower().Contains($certhash.ToLower())
`

In my environment this checking always returns false. But if i replace $result.Get(4) to $result.Get(6) and $result.Get(5) to $result.Get(7), similar ps script works correctly.
And in fact i see that in the "netsh show sslcert" output needed data are located on lines 6 and 7, but not 4 and 5. Is it possible that the command output was changed?

**Comment by AmrutaKawade** [comment]
@DenVifer Feel free to raise PR if you know the solution. We highly encourage community contribution.

**Comment by 20shivangi** [comment]
@DenVifer Is the issue still being faced?

**Comment by DenVifer** [comment]
> @DenVifer Is the issue still being faced?

No, the issue is actual. Unfortunately, it's difficult for me to provide a good PR because it seems like the fix requires nice knowledge about cmd commands in different Windows versions. I just see that your code works in one Windows version, but doesn't in another one. But, I don't even know whether we can in principle use the' http show sslcert' command response in the script. Maybe this response may vary at any time and it's not safe to use it in the PowerShell script. I don't have an answer, cause I don't work with PowerShell at all. We need someone who's cool with this.
